src/main/java/com/github/dockerjava/api/model/Node.java

```diff
@@ -8,20 +9,30 @@
  * A node as returned by the /events API, for instance, when Swarm is used.
  */
 @JsonInclude(Include.NON_NULL)
+@JsonIgnoreProperties(ignoreUnknown = true)
 public class Node {
 
     @JsonProperty("Name")
     private String name;
 
-    @JsonProperty("Id")
+    @JsonProperty("ID")
```

Comment:
It may happen that different calls has different internals in the same Node object, then it should be splitted. 
If `Id` exists in docker api, then you can't just remove it.